charts/scrutiny/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+name: scrutiny
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.16.0"
```

Comment:
this should probably match the scrutiny version right? v0.4.x.
We'll probably want to tweak our CI/CD so that new releases update this file automatically. I can handle that in a separate PR.

Reply:
Yep, I honestly left the defaults as a WIP.

I've had problems in the past with keeping this up to date on CI/CD, so if you can we can also strip it altogether. It is not required for it to be present.